packages/lab/src/slider/internal/usePointerDownTrack.ts

```diff
@@ -0,0 +1,33 @@
+import { RefObject, MouseEvent } from "react";
+import { getValue } from "./utils";
+import { SliderChangeHandler, SliderValue } from "../types";
+
+export function usePointerDownTrack(
+  trackRef: RefObject<HTMLDivElement>,
+  min: number,
+  max: number,
+  step: number,
+  value: SliderValue,
+  onChange: SliderChangeHandler | undefined
+) {
+  return {
+    trackProps: {
+      onPointerDown(event: MouseEvent) {
+        //@ts-ignore - React MouseEvent not compatible with global mouse event, causing type error on SliderTrack
```

Comment:
`getValue` could take in specific values, instead of full `event` to get this working

Reply:
do you know why this isn't casing a type error in usePointerDownThumb, line33 - it's using the same getValue function and also passing a pointer event.
When I try passing clientX to the getValue function, it still seems to be passing the event for some reason

Reply:
if you pass `clientX ` instead of the event, the types of events wont conflict